frontend/javascripts/users/rooms/review_requests/insert_pull_info.js

```diff
@@ -0,0 +1,20 @@
+export default class InsertPullInfo {
+  constructor($root) {
+    console.log(0);
+    this.$root = $root;
+    this.bind();
+  }
+
+  async insertPullInfo(e) {
```

Comment:
__[JSHint]__

Class properties must be methods. Expected '(' but instead saw 'insertPullInfo'.


<!-- Comment by [Sider](https://sider.review) -->